### DISCUSSION TITLE
Slowdown potentially caused by upgrade from 9.0 to 9.3

### DISCUSSION THREAD

**Discussion by LorcanSW** [google/or-tools#3320] [CP-SAT questions]
I recently upgraded my version of or-tools from 9.0.9048 to 9.3.10497, and have experienced a ~30% slowdown to get to an optimal solution. I am running the python version. Are there settings I should change that might help with this?

**Comment by lperron** [comment]
On 1 example, or an a set of examples ?
Statistically speaking, 1 example is not significant. On a generic type of problems, we should have a look.

**Comment by gregy4** [comment]
A lot happened since 9.0, for example increased used memory between 9.0 and 9.1 as described in #2692.
